hyperhtml.js

```diff
@@ -30,7 +30,7 @@ var hyperHTML = (function (globalDocument) {'use strict';
       notAdopting = false;
       hyperHTML.apply(node, arguments);
       notAdopting = true;
-      return this;
+      return node;
```

Comment:
yeah, there are two major adopt fixes that sneaked in this PR, this one, and the one in case the adopted node doesn't have already the expected list of nodes (as a list of LI inside an OL/UL would do).

Thanks 100% code coverage that let me discover that while testing adopted wires :wink: